config/routes.rb

```diff
@@ -1,7 +1,11 @@
 Rails.application.routes.draw do
   devise_for :users
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
-  root "homes#index"
+  root 'homes#index'
+
+  get '/', to: "bands#index"
+  get '/bands/:id', to: "bands#index"
```

Comment:
I'm a little confused by this routing, it doesn't look like you have an `index` action defined in your regular (non-API) bands controller? If these paths are meant for React components, we want to send them `to` a controller and action that leads to a `div` with an `id` of `app`, like the `homes#index`!